java/org.travlyn.server/src/main/java/org/travlyn/server/api/UserApi.java

```diff
@@ -27,7 +30,9 @@
     @GetMapping(
             value = "/user/{userId}/trips",
             produces = {"application/json"})
-    ResponseEntity<List<Trip>> getTripsByUserId(@ApiParam(value = "ID of the user whose trips are to be returned", required = true, defaultValue = "-1", example = "123") @PathVariable(value = "userId") Long userId);
+    ResponseEntity<List<Trip>> getTripsByUserId(
+            @ApiParam(value = "ID of the user whose trips are to be returned", required = true, defaultValue = "-1", example = "123")
```

Comment:
Why are you removing the UserId Param in trip controller but not here?
Is there a certain reason why we need that here?

Reply:
Because a user might want to fetch a public trip from another user.